services/painting-surface-data-simulator-service/app/routers/simulator_router.py

```
@@ -0,0 +1,77 @@
+
+        if not os.path.exists(log_file_path):
+            return {"logs": [], "message": "로그 파일이 없습니다."}
+
+        # 최근 10개 로그만 반환
+        logs = []
+        with open(log_file_path, "r", encoding="utf-8") as f:
+            lines = f.readlines()
+            for line in lines[-10:]:  # 최근 10개
+                try:
+                    logs.append(json.loads(line.strip()))
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)